fMRIVolume/GenericfMRIVolumeProcessingPipeline.sh

```diff
@@ -293,15 +340,61 @@ if [ ! -e "$fMRIFolder" ] ; then
   log_Msg "mkdir ${fMRIFolder}"
   mkdir "$fMRIFolder"
 fi
-cp "$fMRITimeSeries" "$fMRIFolder"/"$OrigTCSName".nii.gz
+${FSLDIR}/bin/imcp "$fMRITimeSeries" "$fMRIFolder"/"$OrigTCSName"
+
+# --- Do slice time correction if indicated
+if [ $DoSliceTimeCorrection = "TRUE" ] ; then
+    log_Msg "Running slice timing correction using FSL's 'slicetimer' tool"
+    TR=`${FSLDIR}/bin/fslval ${fMRIName} pixdim4`
```

Comment:
When I suggested using `fsval` the command I provided was just an example.  In this script `${fMRIName}` is not even a variable.
Use `"$fMRIFolder"/"$OrigTCSName"` instead.
Also, we should probably have a `log_Msg` reporting the TR.
`log_Msg "TR: ${TR}"`

Reply:
Done.